incubator/jmeter-suite/tke-jmeter-suite@v2.0-kona-Non-distributed/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: tke-jmeter-suite-fiber
+name: tke-jmeter-suite-kona
```

Comment:
name is conflict with  https://github.com/tkestack/charts/blob/main/incubator/jmeter-suite/tke-jmeter-suite%40v2.0-kona/Chart.yaml